src/frontend/src/apis/bom.api.ts

```diff
@@ -37,6 +37,16 @@ export const createManufacturer = async (name: string) => {
   return data;
 };
 
+/**
+ * Requests to delete a manufacturer.
+ * @param manufacturer The manufacturer to delete
+ * @returns The deleted manufacturer
+ */
+export const deleteManufacturer = async (name: string) => {
+  const { data } = await axios.post(apiUrls.bomDeleteManufacturer(name), { name });
```

Comment:
We dont need to send any additional information in the payload since it is included in the url.